content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/accordion.html

```diff
@@ -0,0 +1,43 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe Systems Incorporated
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<div data-sly-use.accordion="com.adobe.cq.wcm.core.components.models.Accordion"
+     data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
+     class="cmp-accordion"
+     data-cmp-is="accordion">
+    <ol data-sly-test="${accordion.items && accordion.items.size > 0}"
+        class="cmp-accordion__tablist"
+        role="tablist"
+        aria-multiselectable="false">
+        <li data-sly-repeat.item="${accordion.items}"
+            class="cmp-accordion__item">
+            <div class="cmp-accordion__item--title${item.name == accordion.expandedItem ? ' initially-expanded' : ''}"
+                 role="tab"
+                 data-cmp-hook-accordion="item">
+                ${item.title}
+                <span class="cmp-accordion__item--title--icon"></span>
+            </div>
+
+            <div data-sly-resource="${item.name @ decorationTagName='div'}"
```

Comment:
Right now the content of accordion is not reachable via tab sequence, maybe there should be a tabindex=0 specified?

Reply:
I think this still applies after the accessibility changes

Reply:
I don't think we want the Accordion Item's actual expandable/collapsable contents to be accessible via the tab key. I was under the impression that only links and clickable 'areas' should be accessible by the tab key, right? I could be wrong because I am by no means an accessibility expert, but that was my assumption.

Reply:
I agree here, if there's a focusable element in the item panel then it will naturally gain focus. I don't think we don't want focus to shift to the panel if there's nothing focusable there, but I'm also no accessibility expert.

Reply:
Sorry guys, you’re right, thought that was said in the w3c recomendations, but it was for the tabs component and Not accordion.